### ISSUE TITLE
Cannot find 'winax' module under Windows.

### ISSUE THREAD

**Issue by Najsztub** [poidstotal/stataRun#4]
Hey,

I wanted to run my Stata code using the extension under Win7 and I cannot do this due to the lack of 'winax' extension. Any clue how to get over this?
Below is the error log:
![runstata_error](https://user-images.githubusercontent.com/9817644/45610713-35139200-ba5d-11e8-86b3-81c892e8724f.png)

Best,
Mateusz

**Comment by poidstotal** [comment]
Hi, this is because the extension has declare winax as platform specific requirement, One way would be to go into the terminal and install winax manually. Not sure if that will work, but just an idea. I will try on windows and get back to you. If you re convient with coding you may also want to fix and commit.

**Comment by Najsztub** [comment]
Thanks for the reply.

I've tried to install `winax` manually. After installing Node.js and `npm` I installed `node-gyp`. Unfortunately `node-gyp` does not support Python 3 ( seriously?!). I had to install Python 2.7 together with MS VC++ to get node-gyp running (I've run `npm install --global --production windows-build-tools` from `node-gyp` readme). Unfortunately while trying to run node-gyp on node-activex repo I got an error. I had to reinstall Node. It didn't help.

So for the while I'm stuck with Stata's default editor.
